web-service-consumer/1.6/modules/ROOT/pages/web-service-consumer-troubleshooting.adoc

```diff
@@ -0,0 +1,166 @@
+= Troubleshoot Web Service Consumer Connector - Mule 4
+
+To troubleshoot Anypoint Connector for Web Service Consumer (Web Service Consumer Connector), review the following sections for performing general troubleshooting, addressing specific common errors, and interpreting commonly thrown messages.
+
+== Enable Wire Logging
+
+When troubleshooting Web Service Consumer Connector it is helpful to look at the transport request and response on the Web Service Consumer.
+If you are using the underlying HTTP transport, you can enable wire logging to see the exact HTTP message, the headers used in the transmission and the status of the response.
+
+To enable wire logging:
+
+. Access Anypoint Studio, navigate to the *Package Explorer* view.
+. Open your application's project name.
+. Open the *src/main/resources* path folder.
+. Open the `log4j2.xml` file inside the folder.
+. Add or uncomment the following line:
```

Comment:
By this do we mean that if this line is already there, uncomment it to enable it; otherwise, add it?
I was slowed down a bit by "add or uncomment"...but maybe that is just me?